R/checkDesign.r

```diff
@@ -129,7 +129,23 @@ checkDesign <- function(dat, booklets, blocks, rotation, sysMis = "NA", id = "ID
                      {.envvar {toOmit}}",
                      wrap = TRUE)
     }
-    dat <- dat[, -match(toOmit, names(dat))]
+    dat <- dat[,-c(which( names(dat) %in% toOmit))]
+  }
+
+  # Check for variables in the info (`blocks$subunit`) that are not in `dat`
+  toOmit2 <- setdiff(c(id, blocks$subunit), names(dat))
+  nToOmit2 <- length(toOmit2)
```

Comment:
Nit: Maybe, there is another option for naming both `toOmit` objects, now? 🤔 E.g., (my naive suggestion would be) `toOmitDat` and `toOmitBlocks` (and `nToOmitDat` and `nToOmitBlocks`) or, maybe, just `toOmitDat` and `toOmit`? That might make it easier to identify them.

Reply:
implemented!